### PR TITLE
Added NL translations

### PR DESCRIPTION
**Pull request by SuperDJ**

### CODE REVIEW

File: locales/nl/nl.json [hunk 1]
```diff
@@ -159,8 +159,8 @@
     "Cuba": "Cuba",
     "Curaçao": "Curaçao",
     "Current Password": "Huidig wachtwoord",
-    "Current Subscription Plan": "Current Subscription Plan",
-    "Currently Subscribed": "Currently Subscribed",
+    "Current Subscription Plan": "Huidig abonnement",
+    "Currently Subscribed": "Huidig abonnement",
```
Comment (caouecs) [inline locales/nl/nl.json:163]: No diff between "Current Subscription Plan" and "Currently Subscribed" ?

File: locales/nl/nl.json [hunk 2]
```diff
@@ -250,16 +250,16 @@
...
+    "Heard Island & Mcdonald Islands": "Heard Eiland & McDonald Eilanden",
+    "Heard Island and McDonald Islands": "Heard Eiland en McDonald Eilanden",
     "Hello!": "Hallo!",
     "Hide Content": "Inhoud Verbergen",
     "Hold Up!": "Wacht Even!",
-    "Holy See (Vatican City State)": "Vatican City",
+    "Holy See (Vatican City State)": "Vaticaanstad",
     "Honduras": "Honduras",
-    "Hong Kong": "Hong",
+    "Hong Kong": "Hong Kong",
```
Comment (caouecs) [inline locales/nl/nl.json:262]: If it's same translation, it's for exclude/nl.php

File: locales/nl/nl.json [hunk 3]
```diff
@@ -485,25 +485,25 @@
...
     "Saint Helena": "Sint-Helena",
-    "Saint Kitts and Nevis": "Saint Kitts and Nevis",
+    "Saint Kitts and Nevis": "Saint Kitts en Nevis",
     "Saint Kitts And Nevis": "St. Kitts En Nevis",
     "Saint Lucia": "Saint Lucia",
     "Saint Martin": "St. Martin",
     "Saint Martin (French part)": "Saint Martin (French part)",
     "Saint Pierre and Miquelon": "Saint Pierre and Miquelon",
-    "Saint Pierre And Miquelon": "Min. orde: 1set \/ Sets",
+    "Saint Pierre And Miquelon": "Saint Pierre And Miquelon",
```
Comment (caouecs) [inline locales/nl/nl.json:494]: you don't translate "and" ?

**Review by caouecs — CHANGES_REQUESTED** [review]
Hello,
Dankjewel

just some questions

File: locales/nl/nl.json (hunk 2 above)
Comment (SuperDJ) [reply locales/nl/nl.json:262]: Should be solved in https://github.com/Laravel-Lang/lang/pull/1733/commits/26f0926779cbdd1cac78503d99940599f81c7178

File: locales/nl/nl.json (hunk 3 above)
Comment (SuperDJ) [reply locales/nl/nl.json]: Should be solved in https://github.com/Laravel-Lang/lang/pull/1733/commits/26f0926779cbdd1cac78503d99940599f81c7178

**Comment by caouecs** [comment]
Dankjewel

**Pull request merged**